composer/algorithms/cutmix/README.md

```diff
@@ -101,7 +102,7 @@ CutMix is intended to improve generalization performance, and we empirically fou
 
 > 🚧 Composing Regularization Methods
 >
-> As general rule, composing regularization methods may lead to diminishing returns in quality improvements. CutMix is one such regularization method.
+> As general rule, composing regularization methods may lead to diminishing returns in quality improvements. CutMix is one such regularization method. We do not see improvements when combining CutMix with other regularization and augmentation methods such as Mixup and Label Smoothing.
```

Comment:
I don't think this is true? Ex: over the vanilla ResNet50 baseline, I think CutMix + Label Smoothing gives an improvement. And although we can't do it, it's pretty common to use CutMix + Mixup together in other recipes, alternating between the two, which is supposed to give an improvement